src/components/PostForm/components/TextArea/TextArea.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import { fireEvent, render, screen } from '@testing-library/react'
+
+import { PostFieldProps } from '../../FieldRenderer'
+
+import { TextArea } from '.'
+
+describe('TextArea', () => {
+  const fieldProps: Omit<PostFieldProps, 'onChange' | 'onBlur'> = {
+    field: {
+      key: 'textareaString',
+      title: 'Text Area',
+      type: 'string',
+      directive: 'textarea',
+      minimum: 0,
+      maximum: 1
+    },
+    value: 'default value'
+  }
+
+  it('should render all elements properly', () => {
+    const noop = jest.fn()
+    render(<TextArea {...fieldProps} onChange={noop} onBlur={noop} />)
+    expect(screen.getByLabelText('Text Area')).toBeInTheDocument()
```

Comment:
Just a small change here. From my understanding it's good practice to try to use getByRole wherever possible. Use getByLabelText when you are unable to.

[Priority](https://testing-library.com/docs/queries/about/#priority)

```suggestion
    expect(
      screen.getByRole('textbox', { name: 'Text Area' })
    ).toBeInTheDocument()
```